coffee_core/src/coffee.rs

```diff
@@ -297,6 +297,21 @@ impl PluginManager for CoffeeManager {
         self.storage.store(&self.storage_info()).await?;
         Ok(())
     }
+
+    async fn show(&mut self, plugin: &str) -> Result<Value, CoffeeError> {
+        for repo in &self.repos {
+            if let Some(plugin) = repo.get_plugin_by_name(plugin) {
+                let readme_path = format!("{}/README.md", plugin.path);
```

Comment:
```suggestion
                let readme_path = format!("{}/README.md", plugin.path);
```

we should also consider other options here like README only, but we can do later

Reply:
Yes sure. Is there any other option other than `README` and `README.md`?

Reply:
I think this is the best place to put a `// FIXME: there are more README file options?`